### PR TITLE
[PORT][9.0] base multi image

### PR DESCRIPTION
**Pull request by SodexisTeam**
cc @Yajo

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't lose commit history. You have to follow this procedure: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

**Comment by pedrobaeza** [comment]
The git commit message should contain `[MIG]` instead of `[PORT]`

**Comment by atchuthan** [comment]
@pedrobaeza changes done

**Comment by pedrobaeza** [comment]
For information, relevant commit is https://github.com/OCA/server-tools/pull/402/commits/2dfec51c7e7db6fd5f58d76358a386b33e9ed09b

File: base_multi_image/views/image_view.xml [hunk 1]
```diff
@@ -131,12 +133,13 @@
     <field name="name">Multi images</field>
     <field name="res_model">base_multi_image.image</field>
     <field name="view_mode">kanban,tree,form</field>
+    <field name="help">Click to add multiple image</field>
```
Comment (pedrobaeza) [inline base_multi_image/views/image_view.xml]: Better to put "Click to add an image attached to an specific record."

**Comment by pedrobaeza** [comment]
Little remark, but overall :+1: (code review)

**Comment by elicoidal** [comment]
:+1:

**Comment by atchuthan** [comment]
@pedrobaeza Please add "needs review" label for this PR

**Comment by yajo** [comment]
Cool thanks :+1:

**Pull request merged**